Source/GUI/MainWindow.h

```diff
@@ -22,10 +22,11 @@ class MainWindow : public QMainWindow
   void gameChanged();
   void startSeedFinder();
   void resetPredictor();
-  void rerollPredictor();
+  bool rerollPredictor();
```

Comment:
I am very torn here if we should RETURN a bool or pass one by refference with a clear identifier of what it is.  I am leaning to the later because this is just a function called when pressing a button, returning soemthing here is odd here.  Additionally, it doesn't make it clear what that bool is unless you put a comment, but this shouldn't require one imo.  

I propose the following signature:

`void rerollPredictor(bool* hasAnyWantedStarterPrediction = null)`

Only reason I would use a pointer instead of a refference is to allow the default to be null, you don't always care about that.  It does mean you have to add a check in the body.